### PR TITLE
feat(spanner/spannertest): add support for bool parameter types

### PR DESCRIPTION
**Pull request by sporkmonger**
I tested this against my own codebase as well and it resolved the issue for both my code as well as the integration tests. Previously this would fail with:

```
spanner: code = "Unknown", desc = "rpc error: code = Unknown desc = unsupported well-known type value kind *structpb.Value_BoolValue"
```

### CODE REVIEW

File: spanner/spannertest/integration_test.go [hunk 1]
```diff
@@ -398,6 +398,55 @@ func TestIntegration_SpannerBasics(t *testing.T) {
 	if !reflect.DeepEqual(ages, wantAges) {
 		t.Errorf("Query with IN UNNEST gave wrong ages: got %+v, want %+v", ages, wantAges)
 	}
+
+	// Verify boolean query parameters
```
Comment (dsymonds) [inline spanner/spannertest/integration_test.go:402]: please revert the `integration_test.go` changes. the other tests are sufficient.

**Comment by dsymonds** [comment]
Thanks, this is just an oversight (or having not gotten to it, as a nearby TODO indicates). The fix looks good, but please drop the integration_test.go change.

**Comment by sporkmonger** [comment]
Cool, should be all set.

**Comment by dsymonds** [comment]
Hey @sporkmonger, I think you need to rebase your branch or merge master into it so we can merge this into master.

**Comment by sporkmonger** [comment]
Done.

**Comment by dsymonds** [comment]
Hey @skuruppu, do you know why kokoro isn't reporting back? It's blocking being able to submit this change.

**Comment by skuruppu** [comment]
Yeah, somehow the notification never makes it to GitHub even though the test has run on Kokoro. I usually fiddle with TestFusion to get it working again but I think you need permissions for that.

**Pull request merged**